src/nodejs/analytics/analytics.cpp

```diff
@@ -456,8 +494,23 @@ void TNodeJsSVC::fit(const v8::FunctionCallbackInfo<v8::Value>& Args) {
                 JsModel->Model = TSvm::TLinModel(SvmModel->GetWgtV(), SvmModel->GetThresh());
             }
             else if (JsModel->Algorithm == "LIBSVM") {
-                JsModel->Model = TSvm::LibSvmSolveClassify(VecV, ClsV, JsModel->SvmCost, JsModel->SvmUnbalance,
-                    TQm::TEnv::Debug, TQm::TEnv::Error);
+              int type;
```

Comment:
Parameter parsing does not really belong here and should be done before when we parse out jsons in JsModel. Reason for all the ifs in this function is just to handle different input types (e.g. sparse vs dense matrix) and different algorithms. Rest should be prepared already for them.